drivers/ethernet/eth_mcux.c

```diff
@@ -286,12 +286,14 @@ static void eth_mcux_phy_event(struct eth_context *context)
 					  kENET_MiiReadValidFrame);
 			context->link_up = link_up;
 			context->phy_state = eth_mcux_phy_state_read_duplex;
+			net_if_up(context->iface);
```

Comment:
so following my other comment, that should never be raised by the driver. (link up should, as well as carrier mgmt event btw)

Reply:
So your idea is that we raise yet another event here instead of calling the function directly?